src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
+    private static String[] tasks = new String[100];
+    private static String[] taskStatus = new String[100];
+    private static int numOfTask = 0;
+    //Method to Add to the list of task
+    public static void addTask(String args){
+        numOfTask += 1;
+        tasks[numOfTask-1] = args;
+        taskStatus[numOfTask-1] = " ";
+        System.out.println("added: " + args);
+    }
+
+    public static void listTask(String[] args){
+        for(int i = 1; i <= numOfTask; i+= 1){
+            System.out.println(i + "." + "[" + taskStatus[i-1] + "] "+ args[i-1]);
+        }
+    }
+
+    public static String filter(String sentence){
+        String[] words = sentence.split(" ");
+        for(String word: words){
```

Comment:
Any reason why you used a loop to just check the first word instead of using the first index?